tutor/src/api.coffee

```diff
@@ -172,6 +173,11 @@ start = (bootstrapData) ->
   apiHelper RosterActions, RosterActions.load, RosterActions.loaded, 'GET', (id) ->
     url: "/api/courses/#{id}/roster"
 
+  apiHelper StudentIdActions, StudentIdActions.load, StudentIdActions.loaded, 'GET', (courseId) ->
+    url: "/auth/status"
+  apiHelper StudentIdActions, StudentIdActions.save, StudentIdActions.saved, 'PATCH', (courseId, params) ->
```

Comment:
`/auth/status` is a CC endpoint.  While Tutor can technically use it (obviously), there's not any reason to.  Tutor get's the same data as part of it's bootstrap data, and it's stored in `CurrentUserStore` and `CourseStore`.  

For instance you can get the student ID from there by looking at `CourseStore.get(<courseId>).students`

Something I just realized.   There's no way to find the current student in `students`.  I see you're looking at the first one below in your new `StudentIdStore` but that's not going to work for the case of a teacher who's also a student.  In that case they'll be multiple students in the array.

I think we'll have to add the student id onto the BE's user model so that you can call `CurrentUserStore.getId()` like you can now call `CurrentUserStore.getName()`.  Then use that ID to find the correct student in `CourseStore.get(<courseId>).students`

I'll investigate this on the BE and make a PR there if needed